lilypad/lilypad-v2-testnet/quick-start/README.md

```diff
@@ -0,0 +1,21 @@
+---
+description: Get started with Lilypad Aurora
+---
+
+# Quick Start
+
+The cloud is just somebody else's computer...
+
+<div data-full-width="true">
+
+<figure><img src="../../.gitbook/assets/image (11) (1) (1) (1) (1) (1).png" alt=""><figcaption></figcaption></figure>
```

Comment:
It seems that a lot of the images in here are following the same generic naming format. That said, maybe it would make sense to update all of them to be more descriptive in another PR? Thoughts? @AquiGorka

Reply:
It is good for SEO to treat images correctly (as in give them a proper filename and an accompanying descriptive alt prop), but yes, let's do this in another PR.